tests/test_pipeline/test_autoregressive_pipeline.py

```diff
@@ -37,12 +37,26 @@
 DEFAULT_METRICS = [MAE(mode=MetricAggregationMode.per_segment)]
 
 
-def test_fit(example_tsds):
+@pytest.mark.parametrize("save_ts", [False, True])
```

Comment:
Why do we have such kind of test only here and in Pipeline?

Reply:
I have no idea